install_kind.sh

```diff
@@ -2,6 +2,10 @@
 
 go install sigs.k8s.io/kind@v0.15.0
 
-kind create cluster
+mkdir -p /tmp/kind_storage
```

Comment:
I remember that we also needed to set the permissions to 777, otherwise access from within the container didn't work.